### PR TITLE
added rest of models

### PR DESCRIPTION
**Pull request by timberwang0810**
Added rest of the models

### CODE REVIEW

File: StudyBuddyApp/StudyBuddyApp/Models/Store.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    for (index, element) in self.accessoryItems.enumerated(){
+      if (element == item){
+        self.accessoryItems.remove(at: index)
+        break
+      }
+    }
+    character.onNewItemPurchased(item: item)
+  }
+  
+  public unc purchasePlaygroundItem(item: PlaygroundItem, playground: Playground){
```
Comment (austin-leung) [inline StudyBuddyApp/StudyBuddyApp/Models/Store.swift:42]: func?

File: StudyBuddyApp/StudyBuddyApp/Models/Playground.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }
+  
+  public func moveIntoPlayground(item: PlaygroundItem){
+    for (index, element) in self.storage.enumerated(){
+      if (element == item){
+        self.storage.remove(at: index)
+        break
+      }
+    }
+    if var l = self.decorations[item.category]{
```
Comment (austin-leung) [inline StudyBuddyApp/StudyBuddyApp/Models/Playground.swift:34]: what's l

**Review by austin-leung — APPROVED** [review]
a couple small things

for the record i don't know that most of this stuff works good luck writing tests 💀

File: StudyBuddyApp/StudyBuddyApp/Models/Store.swift (hunk 1 above)
Comment (timberwang0810) [reply StudyBuddyApp/StudyBuddyApp/Models/Store.swift:42]: oop

File: StudyBuddyApp/StudyBuddyApp/Models/Playground.swift (hunk 2 above)
Comment (timberwang0810) [reply StudyBuddyApp/StudyBuddyApp/Models/Playground.swift:34]: an array

**Pull request merged**